Library/Homebrew/sandbox.rb

```diff
@@ -0,0 +1,89 @@
+require "erb"
+require "tempfile"
+
+class Sandbox
+  SANDBOX_EXEC = "/usr/bin/sandbox-exec".freeze
+
+  def self.available?
+    OS.mac? && File.executable?(SANDBOX_EXEC)
+  end
+
+  def initialize(formula=nil)
+    @profile = SandboxProfile.new
+    unless formula.nil?
+      allow_write "/private/tmp", :type => :subpath
```

Comment:
Already added. In last comment, I mean we should add both `HOMEBREW_TEMP` and `/private/tmp`.